content/applications/sales/members/members_analysis.rst

```diff
@@ -0,0 +1,54 @@
+================
+Members analysis
+================
+
+.. |caret| replace:: :icon:`fa-caret-down` :guilabel:`(down)` icon
+.. |pivot| replace:: :icon:`oi-view-pivot` :guilabel:`(pivot)` icon
+.. |list| replace:: :icon:`oi-view-list` :guilabel:`(list)` icon
+
+The *Members Analysis* report provides a detailed overview of all memberships sold and managed
+within the *Members* application.
+
+To access the *Memberships Analysis* report, navigate to :menuselection:`Members app --> Reporting`.
+
+Navigate the report
+===================
+
+The default *Members Analysis* report displays a bar chart, measuring the :guilabel:`Quantity` of
+memberships against the amount of :guilabel:`Revenue Done`. The data is grouped by the assigned
+salesperson, then by the month the membership begins.
+
+.. image:: members_analysis/example-report.png
+   :align: center
+   :alt: A sample of the members analysis report.
+
+Additional :ref:`Group By <search/group>` options include the following:
+
+- :guilabel:`Salesperson`: the internal user in charge of the contact. This information is
+  managed in the :guilabel:`Salesperson` field on the :guilabel:`Sales & Purchase` tab on the
+  contact's record.
+- :guilabel:`Associated Partner`: the contact listed as the *Associated Member* on the
+  :guilabel:`Membership` tab on the contact's record.
+- :guilabel:`Membership Product`: the type of membership assigned to a contact. This includes both
+  purchased memberships and free memberships.
+- :guilabel:`Current Membership State`: the status of the membership.
+- :guilabel:`Month`: the month the membership began.
+
+View results
+============
+
+Click the :icon:`fa-area-chart` :guilabel:`(area chart)` icon to change to graph view. Then, click
```

Comment:
Would "area chart" be universally understood, when the reader is looking for a graph icon? I know that it is the *techinically* correct term to use in the GUI, but it might be easier for *every* reader to know to look for a 'Graph' icon... and also, 'Graph' is what shows up in the tooltip when you hover over that specific icon. Just something to consider during your revisions. 👍